ksql-engine/src/test/java/io/confluent/ksql/function/udf/datetime/TimestampToStringTest.java

```diff
@@ -66,30 +67,53 @@ public void testUTCTimeZone() {
   @Test
   public void testPSTTimeZone() {
     // When:
-    final Object result = udf.timestampToString(1534353043000L,
+    final String result = udf.timestampToString(1534353043000L,
         "yyyy-MM-dd HH:mm:ss", "America/Los_Angeles");
 
     // Then:
     assertThat(result, is("2018-08-15 10:10:43"));
   }
 
   @Test
-  public void testTimeZoneInFormat() {
-    // When:
+  public void testTimeZoneInLocalTime() {
+    // Given:
     final long timestamp = 1534353043000L;
-    final Object localTime = udf.timestampToString(timestamp,
-        "yyyy-MM-dd HH:mm:ss zz");
-    final Object pacificTime = udf.timestampToString(timestamp,
-        "yyyy-MM-dd HH:mm:ss zz", "America/Los_Angeles");
-    final Object universalTime = udf.timestampToString(timestamp,
-        "yyyy-MM-dd HH:mm:ss zz", "UTC");
+
+    // When:
+    final String localTime = udf.timestampToString(timestamp, "yyyy-MM-dd HH:mm:ss zz");
 
     // Then:
     final String expected = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss zz")
         .format(new Date(timestamp));
 
     assertThat(localTime, is(expected));
-    assertThat(pacificTime, is("2018-08-15 10:10:43 PDT"));
+  }
+
+  @Test
+  public void testTimeZoneInPacificTime() {
+    // Given:
+    final long timestamp = 1534353043000L;
+
+    // When:
+    final String pacificTime = udf.timestampToString(timestamp,
+        "yyyy-MM-dd HH:mm:ss zz", "America/Los_Angeles");
+
+    // Then:
+    assertThat(pacificTime,
+        either(is("2018-08-15 10:10:43 PDT"))           // Java 8 and below.
```

Comment:
KSQL should probably standardize on UTC offsets (using the `Z` flag in DateTimeFormatter) or even better switch to ISO-8601 representation. Timezone short-names can be ambiguous, e.g. CST can mean China Standard time or Central Standard time.